pennylane/templates/layers/strongly_entangling.py

```diff
@@ -67,6 +67,7 @@ class StronglyEntanglingLayers(Operation):
     num_params = 1
     num_wires = AnyWires
     par_domain = "A"
+    grad_method = None
```

Comment:
Re. your comment in the PR description about grad methods,

> This operation supports finite-diff but not parameter-shift. Decompose it for parameter-shift support.

Could we make this parameter an ordered list of grad methods, or tuples with grad methods and additional info? For example, `(grad_method, requires_decomposition_to_do_grad_method)`?

Reply:
Yes definitely! The `grad_method` and `grad_recipe` attributes are long overdue for an overhaul. I believe their on the agenda as part of the operator refactor